gov/politeia/proposals.go

```diff
@@ -141,19 +141,30 @@ func generateCustomID(title string) (string, error) {
 	return reg.ReplaceAllString(strings.ToLower(title), "-"), nil
 }
 
-// saveProposals adds the proposals data to the db.
-func (db *ProposalDB) saveProposals(URLParams string) (int, error) {
+// fetchAPIData returns the API data fetched from the Politeia API endpoints.
+// NB: "/api/v1/proposals/vetted" path returns the latest snapshot of the API
+// data that currently exits. This implies that parameter "after=" should be used
+// when syncing the data from scratch and "before=" should be used to fetch newer
+// updates.
```

Comment:
Bug 2:

Reply:
I recall a fix for the before/after semantics already.  Is this the same issue, just in a different part of the gov code, or?

Reply:
It is an addition to that fix.

This bug will be noticed when one does a fresh sync on proposals.db (storm db) file otherwise on normal updates where only the latest data is fetched, all works well.

The actual description highlighted above doesn't exist in the [docs](https://github.com/decred/politeia/blob/master/politeiawww/api/www/v1/api.md#vetted), in fact lack of that description is the reason why we had that previous issue.